.circleci/config.yml

```diff
@@ -149,7 +153,10 @@ commands:
               run_reason='pull request'
             fi
 
-            conbench cpp-micro --run-name="$run_name" --run-reason="$run_reason"
+            conbench cpp-micro \
```

Comment:
are these strings fixed? Could we have the API here being something like

> conbench cpp-micro ...

> conbench upload-results ..

Reply:
The `cpp-micro` string is set here:

https://github.com/facebookincubator/velox/blob/d0868a3b80cb236d7878f6594ef4b81c27069626/scripts/veloxbench/veloxbench/cpp_micro_benchmarks.py#L63

I'll take another look at this too.